hummingbot/user/user_balances.py

```diff
@@ -131,9 +136,12 @@ def validate_ethereum_wallet() -> Optional[str]:
 
     @staticmethod
     def base_amount_ratio(trading_pair, balances):
```

Comment:
This file needs some return type annotations in general. e.g. this function needs `-> Optional[float]`. Please also add return types for `ethereum_balance()`, `balances()`, `all_balances_all_exchanges()`.

Reply:
Done.